server/ai_live_video.go

```diff
@@ -137,6 +137,8 @@ func startTrickleSubscribe(ctx context.Context, url *url.URL, params aiRequestPa
 	// read segments from trickle subscription
 	go func() {
 		var err error
+		firstSegment := false
```

Comment:
```suggestion
		firstSegment := true
```

I think it needs to be `true` first.

Reply:
🤦🏻‍♂️ right